parser/parser.go

```diff
@@ -1075,46 +1077,86 @@ func (p *Parser) parseFormatStringOperator() (token.Token, string, string, error
 	}
 
 	maxTextLength := p.maxLineLength
+	numLines := p.numLines
+	cursorOverlapWidth := p.cursorOverlapWidth
+	paramValue := ""
+
+	namedParameters := map[string]func(){
+		"fontId": func() {
+			fontID = paramValue
+			fontIdToken = p.peekToken
+		},
+		"maxLineLength": func() {
+			num, _ := strconv.ParseInt(paramValue, 0, 64)
+			maxTextLength = int(num)
+		},
+		"numLines": func() {
+			num, _ := strconv.ParseInt(paramValue, 0, 64)
+			numLines = int(num)
+		},
+		"cursorOverlapWidth": func() {
+			num, _ := strconv.ParseInt(paramValue, 0, 64)
+			cursorOverlapWidth = int(num)
+		},
+	}
```

Comment:
We'll also want to raise an error if the user specifies multiple values for the same param.  For example:
```
format("whatever", fontId="latin_1", fontId="latin_2") // Error!
```

Reply:
https://github.com/huderlem/poryscript/pull/59/commits/b834f9d1d0eb65d6d9bb59c96f6be07aa8515bdc